app/scripts/lib/util.js

```diff
@@ -148,6 +148,14 @@ function getRandomArrayItem (array) {
   return array[Math.floor((Math.random() * array.length))]
 }
 
+function mapObjectValues (object, cb) {
```

Comment:
Nit: I think this could be accomplished more simply by using `Object.values(object).map( ... )`. I'm not entirely sure why `object` is deep-cloned via `JSON.stringify` either? That was unexpected to me, from looking at this function name. It looks like the single place this is used doesn't need it to be cloned anyway.

Reply:
`Object.values(object).map( ... )` would return an array, whereas this method returns an object. In particular, it returns an objects with the same keys as the passed object, but modified values, without modifying the original object.

I suppose this could be accomplished with an `Object.entries(object).reduce(...)`

Reply:
Ah I see what you mean now. `mapObjectValues` does seem useful 🤔 
The clone still throws me for a loop though. Seems heavy handed. 🤷‍♂

Reply:
Made it less heavy handed here: c09c4a19b

Reply:
that commit was actually broken, updated commit is here d22c706a6